Assets/Scripts/Core/Movement/Data/DirectionalMoverData.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using Core.Enum;
+using UnityEngine;
+
+namespace Core.Movement.Data
+{
+    [Serializable]
+    public class DirectionalMoverData
+    {
```

Comment:
For what do you need this empty class? Maybe this is a preparation for future functionality, but I don't like this approach. There should be nothing in the code that is not used, as this can cause a lot of questions from other developers